Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.7
+RUN apt-get update -qq && apt-get install -y
```

Comment:
What does this line do?

Reply:
I remove it as this updates should happen in the original image